csrc/preprocess/transform/pad.cpp

```diff
@@ -31,43 +41,36 @@ Result<Value> PadImpl::Process(const Value& input) {
   MMDEPLOY_DEBUG("input: {}", to_json(input).dump(2));
   Value output = input;
   auto img_fields = GetImageFields(input);
-
   for (auto& key : img_fields) {
     Tensor output_tensor;
     auto tensor = input[key].get<Tensor>();
-
     assert(tensor.desc().shape.size() == 4);
     assert(tensor.desc().shape[0] == 1);
     assert(tensor.desc().shape[3] == 3 || tensor.desc().shape[3] == 1);
 
     int height = tensor.desc().shape[1];
     int width = tensor.desc().shape[2];
-
     if (arg_.pad_to_square) {
       int max_size = std::max(tensor.desc().shape[1], tensor.desc().shape[2]);
       std::array padding{0, 0, max_size - width, max_size - height};
 
       OUTCOME_TRY(output_tensor, PadImage(tensor, padding));
-
       output["pad_fixed_size"].push_back(max_size);
       output["pad_fixed_size"].push_back(max_size);
-    } else if (arg_.size_divisor != 1) {
+    } else if (arg_.size[0] != 0 && arg_.size[1] != 0) {
```

Comment:
Please refer to https://github.com/open-mmlab/mmdetection/blob/2733aa961ae98f635c4336e9d88a6f1428b958f9/mmdet/datasets/pipelines/transforms.py#L614